packages/containers/src/ComponentForm/ComponentForm.test.js

```diff
@@ -117,7 +117,7 @@ describe('ComponentForm', () => {
 			// then
 			expect(properties.my.awesome).toEqual({
 				value: ['correct value', 'neither'],
-				$value_name: ['Yes this is the name', 'Neither this one'],
+				$value_name: ['Neither this one', 'Yes this is the name'],
```

Comment:
why value name is reversed compared to the value ?

Reply:
Yes you are right. Good catch.